app/server/tests/configReader.test.ts

```diff
@@ -6,6 +6,12 @@ describe("configReader", () => {
         jest.resetAllMocks();
     });
 
+    it("copes with BOM markers", () => {
+        // This test needs to go before the mocks below
```

Comment:
If you change line 6 to `jest.restoreAllMocks();`. this should no longer be the case!

Reply:
thanks, that does the trick!